generator/src/routes/+page.svelte

```diff
@@ -1,59 +1,246 @@
-<script>
-  import Counter from "./Counter.svelte"
-  import welcome from "$lib/images/svelte-welcome.webp"
-  import welcome_fallback from "$lib/images/svelte-welcome.png"
+<script lang="ts">
+  import Tooltip from "sv-tooltip"
+  import FormInputGroup from "$components/FormInputGroup/index.svelte"
+  import FormSelectGroup from "$components/FormSelectGroup/index.svelte"
+  import FormCheckbox from "$components/FormCheckbox/index.svelte"
+  import Button from "$components/Button/index.svelte"
+  import Toggle from "$components/Toggle/index.svelte"
+  import logoTagsSvg from "$assets/ioxio-tags-logo.svg?url"
+  import effectSvg from "$assets/effect.svg?url"
+  import logomarkSvg from "$assets/ioxio-logomark.svg?url"
+  import logoSvg from "$assets/ioxio-logo.svg?url"
+  import QuestionSvg from "$assets/question.svg?component"
+  import subtractSvg from "$assets/subtract.png"
```

Comment:
I would keep the first letter capitalized for all of these. Also it says `substractSvg`, but it is a `.png` -> please replace with `.svg` file :smile: